storage-stackdriver/src/main/java/zipkin2/storage/stackdriver/StackdriverSpanConsumer.java

```diff
@@ -82,17 +81,21 @@ public BatchWriteSpansCall clone() {
     }
 
     @Override protected void doEnqueue(Callback<Empty> callback) {
-      Futures.addCallback(sendRequest(),
-          new FutureCallback<Empty>() {
-            @Override public void onSuccess(Empty empty) {
-              callback.onSuccess(empty);
-            }
-
-            @Override public void onFailure(Throwable throwable) {
-              callback.onError(throwable);
-            }
-          },
-          MoreExecutors.directExecutor());
+      try {
+        Futures.addCallback(sendRequest(),
+            new FutureCallback<Empty>() {
+              @Override public void onSuccess(Empty empty) {
+                callback.onSuccess(empty);
+              }
+
+              @Override public void onFailure(Throwable throwable) {
+                callback.onError(throwable);
+              }
+            },
+            MoreExecutors.directExecutor());
+      } catch (RuntimeException | Error e) {
```

Comment:
I guess this is equivalent to `Throwable`

Reply:
oops meant to throw this e heh. in such case throwable would jank the signature